CommunityProject/Assets/_Game/Scripts/Runtime/UI/ViewBuildNumber.cs

```diff
@@ -0,0 +1,35 @@
+using BoundfoxStudios.CommunityProject.Build.BuildManifest;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using System;
+
+namespace BoundfoxStudios.CommunityProject
+{
+    public class ViewBuildNumber : MonoBehaviour
```

Comment:
```suggestion
    [AddComponentMenu(Constants.MenuNames.UI + "/" + nameof(ViewBuildNumber))]
    public class ViewBuildNumber : MonoBehaviour
```

Siehe z.B. LoadingScreen als Beispiel.

Für @ManuelRauber: Muss noch in die Dokumentation rein.